crypto/fingerprint/emoji.go

```diff
@@ -0,0 +1,274 @@
+package fingerprint
+
+import "strings"
+
+func toEmoji(input []byte) string {
+	var b strings.Builder
+	for _, r := range input {
+		b.WriteString(emojiCodeMap[r])
+	}
+	return b.String()
+}
+
+// emojiCodeMap is a mapping from byte to emoji.
+//
+// The mapping is based on https://github.com/emojisum/emojisum with amendments
+// proposed in https://github.com/emojisum/emojisum/issues/37.
```

Comment:
fixed! feel free to pull it in.
I tagged a release `draft+2` for easier referencing. https://github.com/emojisum/emojisum/releases/tag/draft%2B2

Reply:
Done.